bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -57,67 +59,200 @@
      * Signal strength default system wide {@link ChannelType}. Represents signal strength of a device as a number
      * with values 0, 1, 2, 3 or 4, 0 being worst strength and 4 being best strength.
      */
-    public static final ChannelType SYSTEM_CHANNEL_SIGNAL_STRENGTH = new ChannelType(
-            new ChannelTypeUID("system:signal-strength"), false, "Number", "Signal Strength", null, "QualityOfService",
-            null,
-            new StateDescription(BigDecimal.ZERO, new BigDecimal(4), BigDecimal.ONE, null, true,
+    public static final ChannelType SYSTEM_CHANNEL_SIGNAL_STRENGTH = new ChannelTypeBuilder().withBindingId("system")
+            .withChannelTypeId("signal-strength").withItemType("Number").withLabel("Signal Strength")
+            .withCategory("QualityOfService")
+            .withStateDescription(new StateDescription(BigDecimal.ZERO, new BigDecimal(4), BigDecimal.ONE, null, true,
                     Arrays.asList(new StateOption("0", "no signal"), new StateOption("1", "weak"),
                             new StateOption("2", "average"), new StateOption("3", "good"),
-                            new StateOption("4", "excellent"))),
-            null);
+                            new StateOption("4", "excellent"))))
+            .build();
 
     /**
      * Low battery default system wide {@link ChannelType}. Represents a low battery warning with possible values
      * on/off.
      */
-    public static final ChannelType SYSTEM_CHANNEL_LOW_BATTERY = new ChannelType(
-            new ChannelTypeUID("system:low-battery"), false, "Switch", "Low Battery", null, "Battery", null,
-            new StateDescription(null, null, null, null, true, null), null);
+    public static final ChannelType SYSTEM_CHANNEL_LOW_BATTERY = new ChannelTypeBuilder().withBindingId("system")
+            .withChannelTypeId("low-battery").withItemType("Switch").withLabel("Low Battery").withCategory("Battery")
+            .withStateDescription(new StateDescription(null, null, null, null, true, null)).build();
```

Comment:
for the StateDescriptions with a lot of null values you could try the StateDescriptionFramentBuilder and `toStateDescription` on the result. may be shorter.